async/kafka/kafka.go

```diff
@@ -9,11 +9,12 @@ import (
 	"github.com/Shopify/sarama"
 	"github.com/beatlabs/patron/async"
 	"github.com/beatlabs/patron/encoding"
+	patron_json "github.com/beatlabs/patron/encoding/json"
```

Comment:
if there is no package collision, no need to add an alias
```suggestion
	"github.com/beatlabs/patron/encoding/json"
```